plugins/generator-1.19.2/forge-1.19.2/mappings/sounds.yaml

```
@@ -0,0 +1,1324 @@
+block.mud_bricks.place: block.mud_bricks.place
+block.mud_bricks.step: block.mud_bricks.step
+block.muddy_mangrove_roots.break: block.muddy_mangrove_roots.break
+block.muddy_mangrove_roots.fall: block.muddy_mangrove_roots.fall
+block.muddy_mangrove_roots.hit: block.muddy_mangrove_roots.hit
+block.muddy_mangrove_roots.place: block.muddy_mangrove_roots.place
+block.muddy_mangrove_roots.step: block.muddy_mangrove_roots.step
+block.packed_mud.break: block.packed_mud.break
+block.packed_mud.fall: block.packed_mud.fall
+block.packed_mud.hit: block.packed_mud.hit
+block.packed_mud.place: block.packed_mud.place
+block.packed_mud.step: block.packed_mud.step
+block.sculk.spread: block.sculk.spread
+block.sculk.charge: block.sculk.charge
+block.sculk.break: block.sculk.break
+block.sculk.fall: block.sculk.fall
+block.sculk.hit: block.sculk.hit
+block.sculk.place: block.sculk.place
+block.sculk.step: block.sculk.step
+block.sculk_catalyst.bloom: block.sculk_catalyst.bloom
+block.sculk_catalyst.break: block.sculk_catalyst.break
+block.sculk_catalyst.fall: block.sculk_catalyst.fall
+block.sculk_catalyst.hit: block.sculk_catalyst.hit
+block.sculk_catalyst.place: block.sculk_catalyst.place
+block.sculk_catalyst.step: block.sculk_catalyst.step
+block.sculk_shrieker.shriek: block.sculk_shrieker.shriek
+block.sculk_shrieker.break: block.sculk_shrieker.break
+block.sculk_shrieker.fall: block.sculk_shrieker.fall
+block.sculk_shrieker.hit: block.sculk_shrieker.hit
+block.sculk_shrieker.place: block.sculk_shrieker.place
+block.sculk_shrieker.step: block.sculk_shrieker.step
+block.sculk_vein.break: block.sculk_vein.break
+block.sculk_vein.fall: block.sculk_vein.fall
+block.sculk_vein.hit: block.sculk_vein.hit
+block.sculk_vein.place: block.sculk_vein.place
+block.sculk_vein.step: block.sculk_vein.step
+entity.tadpole.death: entity.tadpole.death
+entity.tadpole.flop: entity.tadpole.flop
+entity.tadpole.grow_up: entity.tadpole.grow_up
+entity.tadpole.hurt: entity.tadpole.hurt
+entity.warden.agitated: entity.warden.agitated
+entity.warden.ambient: entity.warden.ambient
+entity.warden.angry: entity.warden.angry
+entity.warden.attack_impact: entity.warden.attack_impact
+entity.warden.death: entity.warden.death
+entity.warden.dig: entity.warden.dig
+entity.warden.emerge: entity.warden.emerge
+entity.warden.heartbeat: entity.warden.heartbeat
+entity.warden.hurt: entity.warden.hurt
+entity.warden.listening: entity.warden.listening
+entity.warden.listening_angry: entity.warden.listening_angry
+entity.warden.nearby_close: entity.warden.nearby_close
+entity.warden.nearby_closer: entity.warden.nearby_closer
+entity.warden.nearby_closest: entity.warden.nearby_closest
+entity.warden.roar: entity.warden.roar
+entity.warden.sniff: entity.warden.sniff
+entity.warden.sonic_boom: entity.warden.sonic_boom
+entity.warden.sonic_charge: entity.warden.sonic_charge
+entity.warden.step: entity.warden.step
+entity.warden.tendril_clicks: entity.warden.tendril_clicks
```

```suggestion
entity.warden.tendril_clicks: entity.warden.tendril_clicks
block.mangrove_roots.break: block.mangrove_roots.break
block.mangrove_roots.fall: block.mangrove_roots.fall
block.mangrove_roots.hit: block.mangrove_roots.hit
block.mangrove_roots.place: block.mangrove_roots.place
block.mangrove_roots.step: block.mangrove_roots.step
```
The following sounds are missing from both this mapping, as well as the data list. This suggestion fixes the mapping, but the same must be done in the data lists.
```
block.mangrove_roots.break
block.mangrove_roots.fall
block.mangrove_roots.hit
block.mangrove_roots.place
block.mangrove_roots.step
```